acm_web_site/apps/business/urls.py

```diff
@@ -0,0 +1,5 @@
+from django.urls import path, include
+
+urlpatterns = [
+    path('web_site/', include('apps.business.web_site.urls')),
```

Comment:
GJ! We must divide `web_site` into multiple apps, but this is a good first step!